Description: Bugfix for FvwmIconMan
Author: Jaimos Skriletz <jaimosskriletz@gmail.com>
Forwarded: https://github.com/fvwmorg/fvwm/commit/b2d9422fcdcedf93078d36cb3d76a1860f7da771
Last-Update: 2020-09-09
--- a/default-config/config
+++ b/default-config/config
@@ -621,8 +621,7 @@ DestroyModuleConfig FvwmIconMan:*
 *FvwmIconMan: UseWinList true
 *FvwmIconMan: ButtonGeometry 120x20
 *FvwmIconMan: ManagerGeometry 1x1
-*FvwmIconMan: Background #003c3c
-*FvwmIconMan: Foreground #ffffff
+*FvwmIconMan: Colorset 10
 *FvwmIconMan: FocusColorset 11
 *FvwmIconMan: IconColorset 14
 *FvwmIconMan: FocusAndSelectColorset 12
--- a/modules/FvwmIconMan/xmanager.c
+++ b/modules/FvwmIconMan/xmanager.c
@@ -1115,7 +1115,9 @@ static void clear_empty_region(WinManage
 		rects[0].x, rects[0].y, rects[0].width, rects[0].height,
 		rects[1].x, rects[1].y, rects[1].width, rects[1].height);
 
-  if (CSET_IS_TRANSPARENT_PR_PURE(man->colorsets[DEFAULT]))
+  int cset = man->colorsets[DEFAULT];
+
+  if (CSET_IS_TRANSPARENT_PR_PURE(cset))
   {
     for(n=0; n < num_rects; n++)
     {
@@ -1127,6 +1129,19 @@ static void clear_empty_region(WinManage
 	    }
     }
   }
+  else if (cset >= 0)
+  {
+    for(n=0; n < num_rects; n++)
+    {
+	    if (rects[n].width > 0 && rects[n].height > 0)
+	    {
+		    SetRectangleBackground(theDisplay, man->theWindow,
+			    rects[n].x, rects[n].y, rects[n].width,
+			    rects[n].height, &Colorset[cset], Pdepth,
+			    man->backContext[DEFAULT]);
+	    }
+    }
+  }
   else
   {
     XFillRectangles(theDisplay, man->theWindow,
