Description: Fix spelling errors in binary and manual pages.
Author: Jaimos Skriletz <jaimosskriletz@gmail.com>
Forwarded: https://github.com/fvwmorg/fvwm/commit/02225f83b8e9443a8e7418a9614e90c31708c428
Last-Update: 2021-04-21
--- a/fvwm/bindings.c
+++ b/fvwm/bindings.c
@@ -577,7 +577,7 @@ static int ParseBinding(
 	{
 		fvwm_msg(
 			WARN, "ParseBinding", "Binding specified AnyModifier"
-			" and other modifers too. Excess modifiers are"
+			" and other modifiers too. Excess modifiers are"
 			" ignored.");
 		modifier = AnyModifier;
 	}
--- a/fvwm/menubindings.c
+++ b/fvwm/menubindings.c
@@ -456,7 +456,7 @@ int menu_binding(
 	{
 		fvwm_msg(
 			WARN, "menu_binding", "Binding specified AnyModifier"
-			" and other modifers too. Excess modifiers are"
+			" and other modifiers too. Excess modifiers are"
 			" ignored.");
 		modifier = AnyModifier;
 	}
--- a/modules/FvwmButtons/FvwmButtons.1.in
+++ b/modules/FvwmButtons/FvwmButtons.1.in
@@ -388,7 +388,7 @@ to center the contents on the button.
 This option specifies that this button will not be considered at
 all when making the initial calculations of button sizes. Useful
 for the odd button that gets just a couple of pixels too large to
-keep in line, and therefor blows up your whole buttonbox. "NoSize"
+keep in line, and therefore blows up your whole buttonbox. "NoSize"
 is equivalent to "Size 0 0".
 
 .IP "Padding \fIwidth height\fP"
--- a/modules/FvwmIconMan/readconfig.c
+++ b/modules/FvwmIconMan/readconfig.c
@@ -681,7 +681,7 @@ Binding *ParseMouseEntry(char *tline)
 
   modifiers_string_to_modmask(modifiers, &mods);
   if((mods & AnyModifier)&&(mods&(~AnyModifier))) {
-    ConsoleMessage("Binding specified AnyModifier and other modifers too. "
+    ConsoleMessage("Binding specified AnyModifier and other modifiers too. "
 		    "Excess modifiers will be ignored.");
   }
 
@@ -736,7 +736,7 @@ static Binding *ParseKeyEntry(char *tlin
 
   modifiers_string_to_modmask(modifiers, &mods);
   if((mods & AnyModifier)&&(mods&(~AnyModifier))) {
-    ConsoleMessage("Binding specified AnyModifier and other modifers too. Excess modifiers will be ignored.");
+    ConsoleMessage("Binding specified AnyModifier and other modifiers too. Excess modifiers will be ignored.");
   }
 
   /*
--- a/doc/commands/WindowList.xml
+++ b/doc/commands/WindowList.xml
@@ -243,7 +243,7 @@ causes fvwm to add a menu title with the
 before each group of windows on the same desk.
 With
 <fvwmopt cmd="WindowList" opt="NoLayer"/>,
-the layer of the window is not diplayed.  The options
+the layer of the window is not displayed.  The options
 <fvwmopt cmd="WindowList" opt="ShowPage"/>,
 <fvwmopt cmd="WindowList" opt="ShowPageX"/> and
 <fvwmopt cmd="WindowList" opt="ShowPageY"/>
